src/plugins/mer/mersdkkitinformation.cpp

```diff
@@ -208,7 +208,7 @@ QWidget *MerSdkKitInformationWidget::buttonWidget() const
 
 void MerSdkKitInformationWidget::handleSdksUpdated()
 {
-    const QList<MerSdk*>& sdks = MerSdkManager::sdks();
+    const QList<MerSdk*> &sdks = MerSdkManager::sdks();
```

Comment:
Code polishing should not be mixed with regular code changes. Better leave unchanged, there are more issues in the legacy code.